doc/source/operations/ceph-management.rst

```diff
@@ -0,0 +1,294 @@
+  additional playbooks
+- ``cephadm-commands-pre.yml`` - Runs Ceph commands before post-deployment
+  configuration (You can set a list of commands at ``cephadm_commands_pre_extra``
+  variable in ``$KAYOBE_CONFIG_PATH/cephadm.yml``)
+- ``cephadm-ec-profiles.yml`` - Defines Ceph EC profiles
+- ``cephadm-crush-rules.yml`` - Defines Ceph crush rules according
+- ``cephadm-pools.yml`` - Defines Ceph pools
+- ``cephadm-keys.yml`` - Defines Ceph users/keys
+- ``cephadm-commands-post.yml`` - Runs Ceph commands after post-deployment
+  configuration (You can set a list of commands at ``cephadm_commands_post_extra``
+  variable in ``$KAYOBE_CONFIG_PATH/cephadm.yml``)
+
+There are also other Ceph playbooks that are not part of ``cephadm.yml``
+
+- ``cephadm-gather-keys.yml`` - Populate ``ceph.conf`` and keyrings in kayobe-config by
+  gathering Ceph configuration and keys
+- ``ceph-enter-maintenance.yml`` - Set Ceph to maintenance mode for storage
+  hosts (Can limit the hosts with ``-l <hostname>``)
+- ``ceph-exit-maintenance.yml`` - Unset Ceph to maintenance mode for storage
+  hosts (Can limit the hosts with ``-l <hostname>``)
+
+Running Ceph commands
+---------------------
+
+Ceph commands are usually run inside a ``cephadm shell`` utility container:
+
+.. code-block:: console
+
+   # From storage host
+   sudo cephadm shell
+
+Operating a cluster requires a keyring with an admin access to be available for Ceph
+commands. Cephadm will copy such keyring to the nodes carrying
+`_admin <https://docs.ceph.com/en/latest/cephadm/host-management/#special-host-labels>`__
+label - present on MON servers by default when using
+`StackHPC Cephadm collection <https://github.com/stackhpc/ansible-collection-cephadm>`__.
+
+Adding a new storage node
+-------------------------
+
+Add a node to a respective group (e.g. osds) and run ``cephadm-deploy.yml``
+playbook.
+
+.. note::
+   To add other node types than osds (mons, mgrs, etc) you need to specify
+   ``-e cephadm_bootstrap=True`` on playbook run.
+
+Removing a storage node
+-----------------------
+
+First drain the node
+
+.. code-block:: console
+
+   # From storage host
+   sudo cephadm shell
+   ceph orch host drain <host>
+
+Once all daemons are removed - you can remove the host:
+
+.. code-block:: console
+
+   # From storage host
+   sudo cephadm shell
+   ceph orch host rm <host>
+
+And then remove the host from inventory (usually in
+``$KAYOBE_CONFIG_PATH/inventory/overcloud``)
+
+Additional options/commands may be found in
+`Host management <https://docs.ceph.com/en/latest/cephadm/host-management/>`_
+
+Replacing failing drive
+-----------------------
+
+A failing drive in a Ceph cluster will cause OSD daemon to crash.
+In this case Ceph will go into `HEALTH_WARN` state.
+Ceph can report details about failed OSDs by running:
+
+.. code-block:: console
+
+   # From storage host
+   sudo cephadm shell
+   ceph health detail
+
+.. note ::
```

Comment:
I may be wrong but I think the spacing is incorrect here
```suggestion
.. note::
```

Reply:
Indeed, still renders correctly / sphinx does not complain